packages/schema/lib/functional-constraints/inputFieldGroupsConstraints.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const _ = require('lodash');
+const jsonschema = require('jsonschema');
+
+const actionTypes = ['triggers', 'searches', 'creates', 'bulkReads'];
+const resourceMethods = ['get', 'list', 'hook', 'search', 'create'];
```

Comment:
@Natay Updated the constraint check and its tests to consider action of type `bulkReads` and the `resources` too.